### ISSUE TITLE
Possible pure Python implementation and Keras support

### ISSUE THREAD

**Issue by DonaldTsang** [davidsandberg/facenet#633]
Is it possible to use this library as a module or input vector for Keras?  
Is it also possible to replace the MatLab script, such that the library is pure Python?  
Link: https://github.com/davidsandberg/facenet/search?l=matlabhttps://github.com/davidsandberg/facenet/search?l=matlab

**Comment by cjekel** [comment]
> Is it possible to use this library as a module or input vector for Keras?
No. I'm assuming you are using a TensorFlow backend Keras? If this is true, it wouldn't be too difiicult to port the TensorFlow code here to Keras.

If I only need the last layer of the facenet mode, I'll run the facenet model to calculate the last vector (export_embeddings.py). Then I'll use that vector in Keras for another model...

The MATLAB scripts are not important. You can do all of the functions of this library in Python.

**Comment by DonaldTsang** [comment]
Python has Tensorflow as well, so it is "pure Python" in a sense. "Pure Keras" is impossible though.
